### DISCUSSION TITLE
Is something like this possible? (Godot 4.3)

### DISCUSSION THREAD

**Discussion by Vexcenot** [dialogic-godot/dialogic#2385] [Q&A]
hard to explain in words so watch this short mockup  i made
https://www.youtube.com/watch?v=PjvKze3RZoE

https://www.youtube.com/watch?v=67YYC7z8cY4

**Comment by Jowan-Spooner** [comment]
@Vexcenot The link doesn't work for me. So I have no idea if this is possible...

**Comment by Vexcenot** [comment]
oh yeah? How about now??

https://www.youtube.com/watch?v=PjvKze3RZoE

https://www.youtube.com/watch?v=67YYC7z8cY4
